app/app.js

```diff
@@ -1,66 +1,19 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const morgan = require('morgan')
-
-const db = require('../database/models')
+const express = require('express');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const routes = require('./routes')
 
 const app = express();
-app.use(bodyParser.json())
-app.use(bodyParser.urlencoded({ extended: false }))
-app.use(morgan('dev'))
-
-app.get('/', (req, res) => res.status(200).send('welcome to books api'))
-
-app.post('/api/users', (req, res) => db.User.create({
-  firstName: req.body.first_name,
-  lastName: req.body.last_name,
-  email: req.body.email,
-})
-  .then(user => res.status(201).send(user))
-  .catch(err => res.status(400).send(err)))
-
-app.get('/api/users/:id', (req, res) => db.User.find({
-  where: { id: req.params.id },
-  include: [
-    {
-      model: db.Book,
-      attributes: ['title'],
-      as: 'AuthoredBooks',
-      through: { attributes: [] },
-    },
-  ],
-})
-  .then(user => res.status(200).send(user))
-  .catch(err => res.status(400).send(err)))
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(morgan('dev'));
 
-app.get('/api/book/:id', (req, res) => db.Book.findOne({
-  where: { id: req.params.id },
-  include: [
-    {
-      model: db.User,
-      attributes: ['firstName', 'lastName'],
-      as: 'Authors',
-      through: { attributes: [] },
-    },
-  ],
-})
-  .then(book => res.status(200).send(book))
-  .catch(err => res.status(400).send(err)))
+routes(app);
 
-app.post('/api/books', (req, res) => db.Book.create({
-  title: req.body.title,
-})
-  .then((book) => {
-    book.setAuthors([1])
-    res.status(201).send(book)
-  })
-  .catch(err => res.status(400).send(err)))
-app.put('/api/books/:id', (req, res) => db.Book.findById(req.params.id)
-  .then((book) => {
-    book.addAuthors([1])
-    res.status(201).send(book)
-  })
-  .catch(err => res.status(400).send(err)))
+app.get('/api/v1/', (req, res) => res.status(200).send('welcome to books api'))
```

Comment:
lines 13 to 17 can also be moved to the routes file, at the end of the file and the behavior will still be the same.

Reply:
Worked on it.